xcode/Subconscious/Shared/Components/Common/Suggestion/SuggestionLabelView.swift

```diff
@@ -35,6 +35,23 @@ struct SuggestionLabelView: View, Equatable {
 
     var body: some View {
         switch suggestion {
+        case let .memo(address, title) where address.isLocal:
+            // Do not show full did address for local content
+            Label(
+                title: {
+                    TitleGroupView(
+                        title: Text(
+                            verbatim: !title.isEmpty ? title : empty
+                        ),
+                        subtitle: Text(
+                            verbatim: address.slug.markup
```

Comment:
We relativize own-sphere content. However, we don't really want to relativize local content, as this would be lossy (we would lose the did part that tells us the content is local).

Instead we make an exception case in the view for displaying local content by slug only.